### ISSUE TITLE
[bug]  semantic-release-monorepo ignores pkgRoot

### ISSUE THREAD

**Issue by eng-dibo** [pmowrer/semantic-release-monorepo#121]
I created a [minimal reproduction](https://github.com/eng-dibo/tmp-semantic-release) which is a monorepo contains only one workspace in `./package' dir

both root and package (the workspace) have exactly the same release.config.js file except that package extends `semantic-release-monorepo`.

both of them use the default configurations and only two plugins
```
module.exports = {
  plugins: [
    "@semantic-release/commit-analyzer",
    ["@semantic-release/npm", { pkgRoot: "dist" }],
  ],
  // extends: ["semantic-release-monorepo"],
};
```
the important part here is that `@semantic-release/npm` uses `{ pkgRoot: "dist" }`

`package` contents are typically the same as the root including `dist` folder, except the package name.

if I run `semantic-release` from the root, i.e without `semantic-release-monorepo` it packs the package as expected, and puts the contents of `pkgRoot` dir in the top-level of the published package.

in contrast, when I run the same command from `package`, i.e with `semantic-release-monorepo`, it ignores the option `pkgRoot` and the packed packages contain the folder `dist` instead of its contents.

**steps for reproduction**
1- clone [the repo](https://github.com/eng-dibo/tmp-semantic-release) and install dependencies
2- remove all local and remote tags if exist
3- switch to the branch `next` 
4- modify root's and package's files and create feat/fix commits
5- run the script `release` from the root and package and see the tarball's contents in both cases

you will see different behavior.

**Comment by eng-dibo** [comment]
anyone here?

**Comment by iknowcss** [comment]
@eng-dibo I'm affected by this as well. Today I tried downgrading my `@semantic-release/npm` version down from `9.0.1` to `8.0.3` and that seemed to fix the problem. Does that fix the issue in your example repo?

**Comment by eng-dibo** [comment]
> @eng-dibo I'm affected by this as well. Today I tried downgrading my `@semantic-release/npm` version down from `9.0.1` to `8.0.3` and that seemed to fix the problem. Does that fix the issue in your example repo?

finally, someone came here!! it seems this repo is not maintained anymore and we need to migrate to another solution

**Comment by eng-dibo** [comment]
confirmed as this issue only occurs in v9, v8 is more stable

**Comment by iknowcss** [comment]
@eng-dibo I just started using this repo, damn I didn't realize there wasn't much (or any?) activity in the last year. @pmowrer are you still actively developing this or have you set it aside temporarily?

**Comment by pmowrer** [comment]
@iknowcss This project is essentially in maintenance mode. The aim is to keep pace with changes to `semantic-release`, which seems to have slowed down significantly in recent years. Haven't had much time to commit to open source lately - kids 🤷

**Comment by iknowcss** [comment]
Yeah fair enough, thanks for your work on it so far anyway. We have a way forward now at least. Take care 🙏

**Comment by eng-dibo** [comment]
> The aim is to keep pace with changes to `semantic-release`

we just need your help to fix this vital issue

**Comment by iknowcss** [comment]
@eng-dibo is it an issue with this project or with semantic-release? semantic-release did a major upgrade so I would guess they've changed some API. It might not be too difficult to fix. If someone like yourself put up a PR to fix it then maybe Patrick could review and merge in between juggling kids?

**Comment by greglbd** [comment]
This issue is actually from semantic-release/npm see  https://github.com/semantic-release/npm/issues/504

and this PR that fixes it https://github.com/semantic-release/npm/pull/531 - the PR however is only for v10 so that will be an issue but the thread related to the issue does have a way to patch semantic-release/npm
